app/presenters/avatar_presenter.rb

```diff
@@ -4,8 +4,8 @@ class AvatarPresenter < BasePresenter
   DEFAULT_IMAGE = ActionController::Base.helpers.image_path('user/default.png')
 
   def base_hash
-    { small: image_url_small || DEFAULT_IMAGE,
-      medium: image_url_medium || DEFAULT_IMAGE,
+    { small: image_url(:thumb_small) || DEFAULT_IMAGE,
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Space inside { detected.